src/BuiltinExtensions/ComfyUIBackend/WorkflowGeneratorSteps.cs

```diff
@@ -1378,6 +1378,42 @@ JArray doMaskShrinkApply(WorkflowGenerator g, JArray imgIn)
                     (t2iModel, model, clip, vae) = g.CreateStandardModelLoader(t2iModel, "Refiner");
                     g.FinalLoadedModel = t2iModel;
                     g.FinalModel = model;
+                    string compat = t2iModel.ModelClass?.CompatClass;
```

Comment:
this could shouldn't be here -- CreateStandardModelLoader should be grabbing the correct VAE